code/modules/research/designs/tool_designs.dm

```diff
@@ -399,6 +400,6 @@
 	materials = list(/datum/material/iron = 2000)
 	build_path = /obj/item/wrench/bolter
 	category = list(
-		RND_CATEGORY_TOOLS + RND_SUBCATEGORY_TOOLS_JANITORIAL
```

Comment:
I did wonder why the service lathe had this tool able to be printed, but I assumed it was some sort of hardcore Janitor tech